src/models/mocks/habit.js

```diff
@@ -43,24 +43,24 @@ const EARLY_RISE_HABIT = createHabit({
     { status: Status.FAILED },
     { status: Status.FAILED },
     { status: Status.FAILED },
-    { status: Status.DONE },
-    { status: Status.DONE }
+    { status: "" },
+    { status: "" }
   ]
 });
 
-const PUSHUPS_HABIT = createHabit({
-  name: "Отжимания",
-  measuringValue: "40раз",
-  stats: [
-    { status: Status.DONE },
-    { status: Status.FAILED },
-    { status: Status.FAILED },
-    { status: Status.DONE },
-    { status: Status.DONE },
-    { status: Status.DONE },
-    { status: Status.DONE }
-  ]
-});
+// const PUSHUPS_HABIT = createHabit({
+//   name: "Отжимания",
+//   measuringValue: "40раз",
+//   stats: [
+//     { status: Status.DONE },
+//     { status: Status.FAILED },
+//     { status: Status.FAILED },
+//     { status: Status.DONE },
+//     { status: Status.DONE },
+//     { status: "" },
+//     { status: "" }
+//   ]
+// });
```

Comment:
If you don't use this habit anymore, then it is better to remove it completely. It will be present in VCS (version control system, Git in your case), so if you needed it then you can bring it back later from the old commit.

Reply:
Don't forget to read articles about code comments I've provided in one of the previous PRs :wink

Reply:
I've been reading those articles and deleted unnecessary code.